WORKSPACE

```diff
@@ -170,21 +170,18 @@ load("//third_party/repositories:repositories.bzl", "repositories")
 jvm_maven_import_external(
     name = "org_typelevel__cats_core",
     artifact = scala_mvn_artifact(
-        "org.typelevel:cats-core:0.9.0",
+        "org.typelevel:cats-core:2.2.0",
         SCALA_MAJOR_VERSION,
     ),
-    artifact_sha256 = "3ca705cba9dc0632e60477d80779006f8c636c0e2e229dda3410a0c314c1ea1d",
+    artifact_sha256 = "d7b6f800b50028c61ebed12763a100f31e900eb2f7a4af0d68f9aaaf19c24dc3",
     server_urls = MAVEN_SERVER_URLS,
 )
 
 repositories(
+    fetch_sources = False,
     for_artifact_ids = [
         # test adding a scala jar:
         "com_twitter__scalding_date",
-        # For testing that we don't include sources jars to the classpath
```

Comment:
It's a misleading leftover from repositories refactoring. Also the last supproted `org_psywerx_hairyfotr__linter` is for 2.12.

Reply:
So maybe I didn't notice that before (in the repositories refactoring) but from the comment it still sounds like we need to test that we don't include source jars to the classpath

Reply:
@liucijus not for this PR can you please check this? We either need to remove `#        "org_typelevel__cats_core"` and move the comment up or we should uncomment this and remove the `jvm_maven_import_extrernal` from above

Reply:
Probably it's ok to stay without this comment as it is the duplicate of https://github.com/bazelbuild/rules_scala/pull/1118/commits/1b141a9f1fce3543040aa2a8a19ab9801c9c295f

Reply:
There are two comments here:
1. `# "org_typelevel__cats_core"` itself is commented out. You didn't move it in to the repositories but you also kept it here commented out. We need one or the other (maybe because it's only for testing it should be kept as is and just remove the comment).
2. The comment about why we need it- That relates to what you linked to and my reply is that it's sort of duplicated but in this context it's if someone wants to understand why we use cats in our repo without searching everywhere. I think it's better but I can live with it.

Reply:
1. Are you looking at the same version? I don't see a comment `# "org_typelevel__cats_core"` line in WORKSPACE file
2. Commented

Reply:
On master, line 183

Reply:
@ittaiz do you expect me to do anything more regarding this thread?

Reply:
Yes. I can merge it and we'll do a followup PR just for it (a bit of a shame but I can live with it). What do you prefer?

Reply:
I prefer merge

Reply:
Merging now. Please followup on this